python/cog/server/http.py

```diff
@@ -274,13 +304,6 @@ async def cancel(prediction_id: str = Path(..., title="Prediction ID")) -> Any:
         else:
             return JSONResponse({}, status_code=200)
 
-    @app.post("/shutdown")
```

Comment:
You can't remove this from here! This will mean that models that _do_ boot successfully don't have a shutdown endpoint.